glean-core/python/glean/_ffi.py

```diff
@@ -43,6 +43,7 @@ def _load_header(path: str) -> str:
 ffi = FFI()
 ffi.cdef(_load_header("glean.h"))
 lib = ffi.dlopen(str(Path(__file__).parent / get_shared_object_filename()))
+lib.glean_enable_logging()
```

Comment:
Makes sense, however, eventually we'll probably want to make this controllable through the Python logging APIs.  https://bugzilla.mozilla.org/show_bug.cgi?id=1600946

Reply:
👍
Though we don't allow configuring this on other platforms either (it _is_ configurable by environment variables and by default shouldn't show logging I think).

I leave it here for now.